src/Serialization/Json/OkContentNormalizer.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Biig\Melodiia\Serialization\Json;
```

Comment:
I don't get why this is in json namespace ?

Reply:
You may have XML serialization, as well as JSON-LD or something else. Maybe this namespace is overkill, maybe not. Idk. For now I think it's ok having it.

Reply:
I get it but a normalizer is the transformation from object to array. Json, xml, csv, etc .. is about encoder/serializer, not normalizer right ?

Reply:
Depending on the output, it may change. (Basically json-ld needs a lot of custom data)

Reply:
This impact some other classes. And my point about json-ld which need custom normalization (as well as json-api format) is still alive. :) It will remain this way for now (don't forget it's alpha state of things, namespaces may change, feel free to contribute in that way, there are some others you miss that are probably less understandable).